### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by ninadarbaidze**
Landing page review

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8,11 +8,16 @@
...
+        "@heroicons/react": "^1.0.6",
+        "framer-motion": "^6.5.1",
+        "heroicons": "^1.0.6",
         "i18next": "^21.8.14",
         "next": "12.2.3",
         "next-i18next": "^11.3.0",
         "react": "18.2.0",
-        "react-dom": "18.2.0"
+        "react-dom": "18.2.0",
+        "react-parallax": "^3.5.1"
```
Comment (GigaGiorgadze) [inline package-lock.json:20]: oh come on now, its not that hard to make with vanila css, do not add uneeded dependancies for no reason, check this: https://codepen.io/HenryZarza/pen/VwYagOO

File: components/shared/LanguageSwitcher/LanguageSwitchBtn.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+const LanguageSwitchBtn = () => {
+  const { locale } = useRouter();
+  const [languageModal, setLanguageModal] = useState<boolean>(false);
+  const [currentLanguage, setCurrentLanguage] = useState(locale);
+
+  const openLanChangeHandler = () => {
+    setLanguageModal((prevState) => !prevState);
+  };
+
```
Comment (GigaGiorgadze) [inline components/shared/LanguageSwitcher/LanguageSwitchBtn.tsx]: one jsx componenet per folder, u should make two folders called LanguageSwitchBtn and languageSwitchBTnss

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (ninadarbaidze) [reply package-lock.json:20]: I did with vanilla in the end, I didn't like that package

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]
name of folder should match name of component, LanguageSwitchBtn and LanguageSwitcher does not match

**Review by GigaGiorgadze — APPROVED** [review]

**Pull request merged**